RELEASENOTES.md

```diff
@@ -3,6 +3,7 @@
 ### Enhancements
 
 - Improved how log entries are displayed in the server adminstration window: The timestamp now displays the date when the entry is not from the current day and the context object gets expanded initially. ([#1131](https://github.com/realm/realm-studio/issues/1131), since v1.20.0)
+- Added a menu item to export data to JSON or a local Realm from the Realm Browser window. ([#1134](https://github.com/realm/realm-studio/pull/1134))
```

Comment:
Perhaps add that when the file is exported as a local Realm it will also be stored optimally without additional free space (aka compressed)

Reply:
That should also close a related issue to exporting compressed version of a realm.

Reply:
@kraenhansen

Reply:
I believe the term is "compacted" and as we are not actually zipping the file or anything.

I committed the following change https://github.com/realm/realm-studio/commit/ac0fec03e0f752ffeb41ea41f48824c839b7c7dd